src/store/swap.ts

```diff
@@ -43,7 +44,7 @@ export const defaultToastProperties: ToastOptions = {
 
 export enum SwapDirection {
   TO_STABLE,
-  TO_ANCHOR,
+  TO_MINTED,
 }
```

Comment:
Minted is the new term for Stable, so this needs to be Stable and Anchor. To help avoid confusion, please use the same terms as the contract (wantMinted / wantAnchor). Also these don't need ALL_CAPS because they're namespaced under a constant. e.g. `WantMinted` / `WantAnchor`

Reply:
Done